test/e2e/data-test.ts

```diff
@@ -0,0 +1,183 @@
+import { EmrCategory } from '../../src/endpoints/category/emr/models/emr.entity';
```

Comment:
Please move the mock data inside the `endpoints/category` and separate them into three files accordingly:
- `emr.mock.data.ts`
- `service.mock.data.ts`
- `specialization.mock.data.ts`

Reply:
done.

and this is screenshot of running integration test after update PR
![image](https://user-images.githubusercontent.com/56469224/163309533-e6fe4f0e-2ff2-40e8-85a3-8ca0f98ebd2e.png)

all test passed